src/ui/edit/about-dialog.c

```diff
@@ -106,6 +92,9 @@ bt_about_dialog_init_ui (const BtAboutDialog * self)
       PACKAGE_VERSION, "website", "http://www.buzztrax.org", "wrap-license",
       TRUE, NULL);
 
+  g_free(copyright);
+  copyright = NULL;
+  
```

Comment:
normally I have the free calls at the end. Also here I don't see how the stack based string will cause problems. This would be a compilated problem were BT_RELEASE_YEAR is set to a super large number. I am fine with using snprintf here though.

Reply:
Stack-based is no trouble (actually I never knew about 'alloca' before looking at this project,) I just used g_strdup_printf because it calculates the length for me, rather than having to store a 'len' variable. I'll use snprintf.